LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -2272,6 +2134,398 @@ console.log(params);
         dialog.show();
     }
 
+    function duplicateWorkflow() {
```

Comment:
All of the below functions need comments on them, including describing the variables needed.

Also, should probably consider adding in some default values to ensure things don't break silently.

Reply:
They have been added.